src/hal/k20/layout.ld

```diff
@@ -1,3 +1,7 @@
+__aeabi_unwind_cpp_pr0 = abort;
+__aeabi_unwind_cpp_pr1 = abort;
+__aeabi_unwind_cpp_pr2 = abort;
+
```

Comment:
This was the invisible fallout on the k20 platform created by #285. It didn't show up until we started compiling the examples in travis builds.